src/algokit/cli/init.py

```diff
@@ -315,15 +325,27 @@ def validate(self, document: prompt_toolkit.document.Document) -> None:
 
 
 def _get_template_url() -> TemplateSource:
-    blessed_templates = _get_blessed_templates()
+    description_prefix = "\n     "
+
     choice_value = questionary.select(
-        "Select a project template: ", choices=[*blessed_templates, "<enter custom url>"]
+        "Select a project template: ",
+        choices=[
+            questionary.Choice(
+                title=[("bold", key), ("", description_prefix + tmpl.description)],
+                value=tmpl,
+            )
+            for key, tmpl in _get_blessed_templates().items()
+        ]
+        + [
+            # note: can't use None as default value, that gets ignored
+            questionary.Choice(
+                title=f"<other>{description_prefix}Enter a custom URL - potentially dangerous!", value=object()
```

Comment:
Should we expand on dangerous? e.g. ` (grants execution privileges on your machine during installation)`

Reply:
We do expand on this at length if they select that option, by printing `_unofficial_template_warning`